src/workerd/api/r2-admin.h

```diff
@@ -8,9 +8,15 @@
 #include <workerd/jsg/jsg.h>
 #include <workerd/api/http.h>
 
+namespace edgeworker::api {
+class R2CrossAccount;
+}
+
 namespace workerd::api::public_beta {
 
 class R2Admin: public jsg::Object {
+  struct friend_tag_t {};
```

Comment:
Nit: A short comment describing the purpose of this would be helpful for future generations.